hyperspy/drawing/mpl_hie.py

```diff
@@ -32,6 +32,7 @@ def plot_signal(self):
         imf.title = self.signal_title + " Signal"
         imf.xaxis, imf.yaxis = self.axes_manager.signal_axes
         imf.plot_colorbar = True
+        imf.scalebar_color = self.scalebar_color
```

Comment:
Like `mpl_he.py`, this line is necessary for Image plots to have the correct color